src/typescript.js

```diff
@@ -3,7 +3,7 @@
  */
 
 module.exports = {
-  extends: ['plugin:import/typescript', require.resolve('./base.js')],
```

Comment:
Any reason why do we remove this? It is required for the linter to work.
See https://www.npmjs.com/package/eslint-plugin-import

Though it seems it should be imported at the end

Reply:
I moved it to line #18 https://github.com/parcelLab/eslint-config/blob/5a39eab356199115b3228bf51f15ede23ca15149/src/typescript.js#L18